app/services/edition_url.rb

```diff
@@ -1,29 +1,12 @@
 # frozen_string_literal: true
 
-class EditionUrl
+class PreviewSecretService
```

Comment:
I assume you mean to delete this file?

Reply:
Oops - thanks for spotting...deleted :-).